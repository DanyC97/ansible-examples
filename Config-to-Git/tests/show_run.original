
Building configuration...

  
Current configuration : 2331 bytes
!
! Last configuration change at 05:37:16 UTC Mon Apr 9 2018 by cisco
!
version 15.6
service timestamps debug datetime msec
service timestamps log datetime msec
no service password-encryption
!
hostname R1
!
ntp clock-period 1234567
!
end
