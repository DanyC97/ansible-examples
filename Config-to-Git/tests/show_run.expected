!
!
version 15.6
service timestamps debug datetime msec
service timestamps log datetime msec
no service password-encryption
!
hostname R1
!
!
end